services/github-service-impl/src/main/java/org/kontinuity/catapult/service/github/impl/kohsuke/KohsukeGitHubServiceImpl.java

```diff
@@ -158,7 +167,25 @@ public GitHubRepository createRepository(String repositoryName,
         return wrapped;
     }
 
-    /**
+   @Override
+   public GitHubRepository createRepository(String repositoryName, String description, File path) throws IllegalArgumentException {
```

Comment:
I think the method name should mention that it pushes to a repository, or maybe we split into different methods:

`public void push(File path, GithubRepository repository)`